W-16898521: Document no-cache directive from Postman when using Cachi… (#681)
mulesoft/docs-gateway: gateway/1.8/modules/ROOT/pages/policies-included-http-caching.adoc

== FAQ

> *I am using Postman or a similar tool and the policy is not caching anything?*

Reviewer: ```suggestion
*I'm using Postman or a similar tool and the policy isn't caching anything?*
```
